### ISSUE TITLE
Release checklist

### ISSUE THREAD

**Issue by kcpevey** [Quansight/numpy-threading-extensions#36]
* [x] create release branch
* [x] setup rever (rever handles updating version, releasing, tagging, history, changelog) https://github.com/Quansight/numpy-threading-extensions/pull/57
  * [x] initialize rever (create yml file)
  * [x] ensure that rever is pointing to the right version tag
  * [x] run rever release command
* [x] set up publishing wheels to pypi (py 3.6, 3.7, 3.8) https://github.com/Quansight/numpy-threading-extensions/pull/51
  * [x] fast_numpy_loops-0.0.6-cp36-cp36m-win_amd64.whl
  * [x] fast_numpy_loops-0.0.6-cp37-cp37m-win_amd64.whl
  * [x] fast_numpy_loops-0.0.6-cp38-cp38-win_amd64.whl
  * [x] fast_numpy_loops-0.0.6-cp36-cp36m-macosx_10_9_x86_64.whl
  * [x] fast_numpy_loops-0.0.6-cp37-cp37m-macosx_10_9_x86_64.whl
  * [x] fast_numpy_loops-0.0.6-cp38-cp38-macosx_10_9_x86_64.whl
  * [x] fast_numpy_loops-0.0.6-cp36-cp36m-manylinux1_x86_64.whl
  * [x] fast_numpy_loops-0.0.6-cp36-cp36m-manylinux2010_x86_64.whl
  * [x] fast_numpy_loops-0.0.6-cp37-cp37m-manylinux1_x86_64.whl
  * [x] fast_numpy_loops-0.0.6-cp37-cp37m-manylinux2010_x86_64.whl
  * [x] fast_numpy_loops-0.0.6-cp38-cp38-manylinux1_x86_64.whl
  * [x] fast_numpy_loops-0.0.6-cp38-cp38-manylinux2010_x86_64.whl
  * [x] fast-numpy-loops-0.0.6.tar.gz
* [ ] documentation
  * [x] update readme - @daletovar  (add contents of index.rst to readme)
  * [x] installation
  * [x] build instructions
  * [ ] what's included in the first release
    * [x] @mattip skeleton. 
    * [ ] @kcpevey fill in the gaps
  * [ ] roadmap for future releases
    * [x] @mattip skeleton
    * [ ] @kcpevey fill in the gaps
  * [ ] use cases (low priority)
    * [x] @mattip skeleton
    * [ ] @daletovar  fill in the gaps
  * [x] sphinx setup
  * [x] benchmarks
    * [x] #49 @mattip skeleton
  * [x] doc strings
  * [x] publication of docs (gh pages)
  * [x] Add Code of Conduct @kcpevey #61 
* [ ] finalize name
  * [x] come up with short list - @rpekrul 
  * [ ] reach out to numpy mailing list - @mattip 
  * [x] identify author and maintainer mailing list - Waiting on identifying the communication channel #58 @rpekrul 
    * author:  https://github.com/rtosholdings
    * name: accelerated_numpy
    * mailing list: (Google Groups) https://groups.google.com/g/accelerated-numpy
  * [x] implement new name
    * [x] code changes - @kcpevey #62
    * [x] pypi project registration - @kcpevey 

LOW PRIORITY:
* [ ] set up publishing to conda forge (__lower priority__) - _Waiting_
* [ ] benchmarks - extend to show threading, visuals, ensure this is discussed in docs (low priority)

BLOCKED:
* win x32 - [BLOCKED](https://github.com/Quansight/numpy-threading-extensions/issues/44)

**Comment by mattip** [comment]
I think we should add a benchmark report to the documentation that includes
- script to reproduce
- textual description
- a bar graph

**Comment by mattip** [comment]
Also we should add a [Code of Conduct](https://github.com/Quansight/numpy-threading-extensions/community/code-of-conduct/new?template=contributor-covenant)

**Comment by mattip** [comment]
docs are now live at https://quansight.github.io/numpy-threading-extensions. They are served from the `site` branch in the `docs` directory. I am working on adding an automatic build/push/update/ to CI

**Comment by kcpevey** [comment]
@mattip I think @daletovar was also working on making the github action.

**Comment by rpekrul** [comment]
@kcpevey @daletovar @mattip Updated list with assignments, let me know how to re-distribute if the assignments need to be adjusted.  Sounds like we are on track.

**Comment by mattip** [comment]
Closing, the 0.0.1 release is out.
